From 5bc1e23411b066de880e047347dcf2da42215a0a Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Tue, 17 Feb 2026 16:29:40 +0000
Subject: [PATCH 5/8] Remove unused GrapheneOS apps from build

Remove apps and APEXes that are not useful for GSI builds:
- Auditor: relies on hardware attestation tied to specific device
  models and will never pass verification on a GSI
- PdfViewerGOS: can easily be installed from the app store
- InfoApp: not relevant for GSI builds
- DeviceDiagnostics: GrapheneOS hardware diagnostics, cannot
  properly detect hardware on a GSI
- EasterEgg: novelty app, not needed
- HardeningTestApp: debug/testing app, not for production
- com.android.virt: pKVM virtualization APEX, requires
  Pixel-specific firmware support not available on GSI targets

Change-Id: I0000000000000000000000000000000000000003
---
 target/product/base_system.mk     | 2 --
 target/product/handheld_product.mk | 3 ---
 target/product/handheld_system.mk  | 2 --
 3 files changed, 7 deletions(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index abc1234..def5678 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -76,7 +76,6 @@ PRODUCT_PACKAGES += \
     com.android.tethering \
     $(RELEASE_PACKAGE_TZDATA_MODULE) \
     com.android.uwb \
-    com.android.virt \
     com.android.wifi \
     ContactsProvider \
     content \
@@ -572,7 +571,6 @@ PRODUCT_PACKAGES += \
     unwind_info \
     unwind_reg_info \
     unwind_symbols \
-    HardeningTestApp \
 
 # The set of packages whose code can be loaded by the system server.
 PRODUCT_SYSTEM_SERVER_APPS += \
diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index dde952e..737f7f2 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -23,17 +23,14 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 # /product packages
 PRODUCT_PACKAGES += \
     AppStore \
-    Auditor \
     Calendar \
     Camera \
     Contacts \
     DeskClock \
     ExactCalculator \
     Gallery2 \
-    InfoApp \
     LatinIME \
     Music \
-    PdfViewerGOS \
     preinstalled-packages-platform-handheld-product.xml \
     SettingsIntelligence \
     ThemePicker \
diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
index abc1234..def5678 100644
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -47,10 +47,8 @@ PRODUCT_PACKAGES += \
     CaptivePortalLogin \
     CertInstaller \
     CredentialManager \
     DeviceAsWebcam \
-    DeviceDiagnostics \
     DocumentsUI \
     DownloadProviderUi \
-    EasterEgg \
     ExternalStorageProvider \
     FusedLocation \
     GmsCompat \
-- 
2.52.0

